llvm/test/DebugInfo/X86/split-dwarf-dwo-hash.ll

```diff
@@ -0,0 +1,33 @@
+; RUN: rm -rf %t && mkdir -p %t
```

Comment:
This is based on the llvm/test/DebugInfo/X86/split-dwarf-multiple-cu-hash.ll file (which git thinks I've deleted). I'm not sure what happened, but that test was not functional right now (CHECK-NOT was matching not because the DWO_ids were different, but because the output contained only one CU)